autotst/reaction.py

```diff
@@ -27,36 +27,29 @@
 #
 ##########################################################################
 
-import os
-import itertools
-import logging
+import os, itertools, logging
 import numpy as np
 from copy import deepcopy
 
 import rdkit
-from rdkit import DistanceGeometry
-from rdkit.Chem import rdDistGeom
-
-
-from rdkit import Chem
-from rdkit.Chem import AllChem
-from rdkit.Chem.Pharm3D import EmbedLib
+import rdkit.Chem 
+import rdkit.Chem.AllChem
+import rdkit.Chem.Pharm3D.EmbedLib
+import rdkit.DistanceGeometry
 
 import ase 
 
 import rmgpy
-from rmgpy.molecule import Molecule as RMGMolecule
-from rmgpy.species import Species as RMGSpecies
-from rmgpy.reaction import Reaction as RMGReaction
-from rmgpy.reaction import ReactionError
-from rmgpy.kinetics import PDepArrhenius, PDepKineticsModel
+import rmgpy.molecule
+import rmgpy.species 
+import rmgpy.reaction 
 from rmgpy.data.rmg import RMGDatabase
```

Comment:
I had just convinced myself that the rule you're following is that anything from another module you always refer to explicitly, eg. this would be referred to as `rmgpy.data.rmg.RMGDatabase` whenever used, and not imported as `RMGDatabase`.

Was I wrong about your rule, or did you just miss this one?

Reply:
Thanks for the comment. I was thinking that RMGDatabase has a unique enough name so I didn't change it, but I feel like it'll help future developers if we specify the full import. Making that change now.